source/catalog.html

```diff
@@ -1 +1,307 @@
+                <dd class="features__description">500 г</dd>
+                <dt class="features__property">Вкус</dt>
+                <dd class="features__description">Рис</dd>
+                <dt class="features__property">Цена</dt>
+                <dd class="features__description">500 Р.</dd>
+              </dl>
+            </div>
+            <a class="card__order-link" href="" aria-label="Заказать продукт">Заказать</a>
+          </li>
+          <li class="catalog-main__item card__last-card">
+            <button class="main-catalog__toggler visually-hidden" type="button">+</button>
+            <h3 class="card__title">Показать еще 100500 товаров</h3>
+            <p class="card__text">На самом деле вкусов гораздо больше!</p>
+            <a class="card__assortment-link" href="#" aria-label="Показать весь ассортимент">Показать все</a>
+          </li>
+        </ul>
+      </section>
+      <section class="catalog-extra">
+        <div class="catalog-extra__wrapper">
+        <h2 class="catalog-extra__title">Дополнительные товары</h2>
+          <ul class="catalog-extra__list">
+            <li class="catalog-extra__item card-extra">
+              <h3 class="card-extra__title">Сахарозаменитель</h3>
+              <dl class="card-extra__features features-extra">
+                <dt class="features-extra__property">1 упаковка (100 г)</dt>
+                <dd class="features-extra__description">200 Р.</dd>
+              </dl>
+              <a class="card-extra__order-link" href="" aria-label="Заказать продукт">Заказать</a>
+            </li>
+            <li class="catalog-extra__item card-extra">
+              <h3 class="card-extra__title">Питьевая вода</h3>
+              <dl class="card-extra__features features-extra">
+                <dt class="features-extra__property">5 литров</dt>
+                <dd class="features-extra__description">50 Р.</dd>
+              </dl>
+              <a class="card-extra__order-link" href="" aria-label="Заказать продукт">Заказать</a>
+            </li>
+            <li class="catalog-extra__item card-extra">
+              <h3 class="card-extra__title">Молоко</h3>
+              <dl class="card-extra__features features-extra">
+                <dt class="features-extra__property">1 литр</dt>
+                <dd class="features-extra__description">100 Р.</dd>
+              </dl>
+              <a class="card-extra__order-link" href="" aria-label="Заказать продукт">Заказать</a>
+            </li>
+            <li class="catalog-extra__item card-extra">
+              <h3 class="card-extra__title">Витамины</h3>
+              <dl class="card-extra__features features-extra">
+                <dt class="features-extra__property">1 упаковка (30 г)</dt>
+                <dd class="features-extra__description">300 Р.</dd>
+              </dl>
+              <a class="card-extra__order-link" href="" aria-label="Заказать продукт">Заказать</a>
```

Comment:
Давай везде заглушки ставить
```suggestion
              <a class="card-extra__order-link" href="#" aria-label="Заказать продукт">Заказать</a>
```